### DISCUSSION TITLE
Template for new issues

### DISCUSSION THREAD

**Discussion by MarcMil** [soot-oss/soot#1620] [Ideas]
There should be standard templates for new Issues.
I think we should take a pretty good look at how apktool does it:
https://github.com/iBotPeaches/Apktool/issues/new/choose

They have various templates containing useful questions by default. Including (for bugs): What is the input (in case of APKTool it's an apk, but for us this can be more generic).
This avoids unnecessary back and forth.

**Comment by linghuiluo** [comment]
yeah, totally agree and also guidelines for PRs

**Reply by linghuiluo** [reply]
made this https://github.com/soot-oss/soot/issues/new/choose

**Reply by MarcMil** [reply]
Looks good.
I'd maybe change "Please include soot options you used" to also indicate that this may include code snippets in case Soot is used as a library instead of a standalone command line options.
Maybe something along the lines of "Please include soot options you used or supply a code snippet to ease reproduction of the problem.", but this is just nitpicking.

**Reply by linghuiluo** [reply]
yeah, that was also what I meant, but your suggestion is clear, adapted now. Thanks!
